pom.xml

```
@@ -57,7 +57,7 @@
     <!-- when changing version also update version in LICENSE_binary  -->
     <hdrhistogram.version>2.1.12</hdrhistogram.version>
     <metrics.version>4.1.18</metrics.version>
-    <netty.version>4.1.119.Final</netty.version>
+    <netty.version>4.1.129.Final</netty.version>
```

130 is the newest version fixing some regressions introduced in 129
```suggestion
    <netty.version>4.1.130.Final</netty.version>
```